### DISCUSSION TITLE
video compression support

### DISCUSSION THREAD

**Discussion by fly1198721171** [InterDigitalInc/CompressAI#108] [Q&A]
when the platform support HEVC and VVC?

**Comment by fracape** [comment]
Hi, noticing this very late. Please check the last version which includes a first model [Agustsson2020] and evaluation pipelines with hybrid codecs (x264, x265 and VVC (VTM software) for now)

**Reply by fly1198721171** [reply]
OK, thanks for your replay！
the codecs whether include HEVC (not x265，its HM16.xx) ?

**Reply by fracape** [reply]
will do that some day yes, like for images
